fiftyone/brain/__init__.py

```diff
@@ -250,11 +252,6 @@ def compute_uniqueness(
         progress (None): whether to render a progress bar (True/False), use the
             default value ``fiftyone.config.show_progress_bars`` (None), or a
             progress callback function to invoke instead
-        similarity_backend (None): the similarity backend to use. The supported values are
```

Comment:
After sitting with this for a bit, I think it is cleaner to expect the user to create the `SimilarityIndex` offline first and pass it in via `similarity_index` if they want control over what index is used to compute uniqueness. One key reason is my comment below about how the embeddings unfortunately need to be downloaded anyway to figure out what samples exist in the index and can therefore have their uniqueness computed.

Reply:
I am in favor of this. Keeps the implementation simpler.